extra-dev/packages/coq-hott/coq-hott.8.10.dev/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "http://homotopytypetheory.org/"
+bug-reports: "https://github.com/HoTT/HoTT/issues"
+license: "BSD 2-clause"
+build: [
+  ["bash" "-c" "./autogen.sh -skip-submodules || :"]
+  ["./configure" "COQBIN=%{bin}%" "--prefix=%{prefix}%"]
+  ["%{make}%" "-j%{jobs}%"]
```

Comment:
```suggestion
  [make "-j%{jobs}%"]
```
Is there a reason for not using the "builtin" OPAM make command here (which supports several make implementations)?